dotnet-trace/test.sh

```diff
@@ -48,15 +48,16 @@ else
    exit 1
 fi 
 
-dotnet-trace report $FILENAME topN
-if [ -f $REPORTNAME ]; then
-   echo "report - OK"
-else 
+output=$(dotnet-trace report $FILENAME topN)
+if [[ $output == *"Missing Symbol"* ]]; then 
    echo "report - FAIL"
    rm -r $PROJNAME
    rm $FILENAME
    rm $SPEEDSCOPENAME
+   rm $REPORTNAME
    exit 1
+else
```

Comment:
Can you validate the expected output in the OK case?

Reply:
Validation has been added.